examples/documentation_examples/examples.go

```diff
@@ -2956,3 +2957,178 @@ func StableAPIExamples() {
 	StableAPINonStrictExample()
 	StableAPIDeprecationErrorsExample()
 }
+
+func insertSnapshotQueryTestData(mt *mtest.T) {
+	catColl := mt.CreateCollection(mtest.Collection{Name: "cats"}, true)
+	_, err := catColl.InsertMany(context.Background(), []interface{}{
+		bson.D{
+			{"adoptable", false},
+			{"name", "Miyagi"},
```

Comment:
Is one of these the name of your cat haha

Reply:
Miyagi is the name the shelter gave my cat, but I call him Mo :)